.travis.yml

```diff
@@ -4,7 +4,6 @@ php:
  - 7.1
  - 7.2
  - 7.3
- - nightly
```

Comment:
Alternatively, we could use `7.4snapshot` here (see for example 
https://github.com/doctrine/annotations/pull/241).